meson_options.txt

```diff
@@ -34,30 +34,30 @@ option(
 option(
   'zstd',
   type : 'feature',
-  value : 'disabled',
-  description : 'Handle Zstandard-compressed modulesi. Default: disabled',
+  value : 'enabled',
+  description : 'Handle Zstandard-compressed modulesi. Default: enabled',
```

Comment:
Pre-existent typo that can be fixed here: modulesi

Reply:
Pushed an extra commit on top. Thanks o/